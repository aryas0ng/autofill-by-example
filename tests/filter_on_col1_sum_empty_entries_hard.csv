2,5,5
4,3,3
8,4,4
6,8,23
,10,
,5,
1,0,
,4,
19,4,
,4,
,21,
,21,
9,10,
7,7,
,4,
3,3,
,3,
,16,
1,16,
,27,
,8,
3,3,
,3,
8,11,
,4,