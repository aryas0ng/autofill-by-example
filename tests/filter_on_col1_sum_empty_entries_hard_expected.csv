2,5,5
4,3,3
8,4,4
6,8,23
6,10,23
6,5,23
1,0,4
1,4,4
19,4,50
19,4,50
19,21,50
19,21,50
9,10,10
7,7,11
7,4,11
3,3,22
3,3,22
3,16,22
1,16,51
1,27,51
1,8,51
3,3,6
3,3,6
8,11,15
8,4,15